src/Icon/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './Icon.scss';
+
+function Icon(props) {
+  return (
+    <div>
+      <span
+        id={props.id}
+        className={classNames(props.className)}
+        aria-hidden={props.hidden}
+      />
+      { props.srText &&
+        <span className={classNames(styles['sr-only'])}>
```

Comment:
is this `classNames` necessary?

Reply:
It was more for uniformity and removing the question of "wait why was this used in the above element but not this span?"